packages/gatsby-transformer-rehype/package.json

```diff
@@ -46,6 +46,6 @@
         "cross-env": "7.0.3"
     },
     "peerDependencies": {
-        "gatsby": "^3.0.0 || ^4.0.0"
+        "gatsby": "^4.0.0 || ^5.0.0"
```

Comment:
is there a reason to drop 3 support?

Reply:
If we want the short validation without "if subplugins defined", we need https://github.com/gatsbyjs/gatsby/commit/3260b1a43ace66349b0a683ab052f0d5c10f33d2 which is part of 4.0.